Create Start-SPOSiteOpticalCharacterRecognitionBackfill.md (#839)
MicrosoftDocs/OfficeDocs-SharePoint-PowerShell: sharepoint/sharepoint-ps/sharepoint-online/Start-SPOSiteOpticalCharacterRecognitionBackfill.md

Initiates a job to trigger the Optical Character Recognition (OCR) process for all files for the selected site.

## SYNTAX

```Powershell

Start-SPOSiteOpticalCharacterRecognitionBackfill [-Site] <string> 
```

## DESCRIPTION

This cmdlet starts a job to trigger the OCR process for files that have either never been OCR processed or have been modified since the last OCR process. This ensures all relevant content in the site is recognized and indexed for improved searchability and accessibility. 

OCR backfill can only be run on OCR-enabled sites. If OCR is disabled for the site, please enable OCR before proceeding with OCR backfilling. Refer to this [article](/en-us/microsoft-365/syntex/ocr) for instructions on enabling OCR on the selected site.

## EXAMPLES

### EXAMPLE 1

```powershell

Start-SPOSiteOpticalCharacterRecognitionBackfill -Site https://contosoenergy.sharepoint.com/sites/hr
```

Starts OCR process for all content that hasn't been processed before in the selected site.

## PARAMETERS

### -Site

Specifies the site URL where OCR process should be enabled on.

```yaml

> Type: String

Reviewer: ```suggestion
Type: SpoSitePipeBind
```
This parameter's type should be SpoSitePipeBind

Parameter Sets: Default
Aliases:
Applicable: SharePoint Online

Required: True
Position: 1
Default value: None

> Accept pipeline input: False

Reviewer: ```suggestion
Accept pipeline input: True
```
The cmdlet allows values from pipeline, so this should be true

---
external help file: sharepointonline.xml
Module Name: Microsoft.Online.SharePoint.PowerShell
online version: https://learn.microsoft.com/powershell/module/sharepoint-online/Start-SPOSiteOpticalCharacterRecognitionBackfill
applicable: SharePoint Online
title: Start-SPOSiteOpticalCharacterRecognitionBackfill
schema: 2.0.0
author: FarreltinF
ms.author: fanyi
ms.reviewer:
---

# Start-SPOSiteOpticalCharacterRecognitionBackfill

## SYNOPSIS

> [!important]

> > Optical Character Recognition (OCR) is a pay-as-you-go feature, triggering this cmdlet will incur cost for your organization

Reviewer 2: The comma is misplaced -- maybe replace it with "and" or "." instead. Also add `.` at end of sentence.

> Initiates a job to trigger the Optical Character Recognition (OCR) process for all files for the selected site.

Reviewer 2: >the Optical Character Recognition (OCR) [](http://example.com/codeflow?start=27&length=39)

the OCR process

## SYNTAX

```Powershell

> Start-SPOSiteOpticalCharacterRecognitionBackfill [-Site] <string> 

Reviewer 2: ><string> [](http://example.com/codeflow?start=57&length=8)

I think it should be `<SpoSitePipeBind>`?

```

## DESCRIPTION

This cmdlet starts a job to trigger the OCR process for files that have either never been OCR processed or have been modified since the last OCR process. This ensures all relevant content in the site is recognized and indexed for improved searchability and accessibility. 

> OCR backfill can only be run on OCR-enabled sites. If OCR is disabled for the site, please enable OCR before proceeding with OCR backfilling. Refer to this [article](/en-us/microsoft-365/syntex/ocr) for instructions on enabling OCR on the selected site.

Reviewer 2: >/en-us [](http://example.com/codeflow?start=166&length=6)

remove locale from the relative links

Reviewer 2: other places down below as well

> [!Note]
> Backfilling cancellation is currently under development. Once the cmdlet is triggered, the backfilling process cannot be stopped. Disabling OCR on the selected site will not cancel the ongoing backfilling session.

## EXAMPLES

### EXAMPLE 1

```powershell

Start-SPOSiteOpticalCharacterRecognitionBackfill -Site https://contosoenergy.sharepoint.com/sites/hr
```

Starts OCR process for all content that hasn't been processed before in the selected site.

## PARAMETERS

### -Site

> Specifies the site URL where OCR process should be enabled on.

Reviewer 2: >site URL [](http://example.com/codeflow?start=14&length=8)

just "site" or "site object", since all these can be used as the param value: site Id, site Url, SPSite object itself

```yaml
Type: SpoSitePipeBind
Parameter Sets: Default
Aliases:
Applicable: SharePoint Online

Required: True
Position: 1
Default value: None
Accept pipeline input: True
Accept wildcard characters: False
```

### CommonParameters

> This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

Reviewer 2: ```suggestion
This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`, `-InformationAction`, `-InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`, `-ProgressAction`, `-Verbose`, `-WarningAction`, and `-WarningVariable`. For more information, see [about_CommonParameters](/powershell/module/microsoft.powershell.core/about/about_commonparameters).
```

---
external help file: sharepointonline.xml
Module Name: Microsoft.Online.SharePoint.PowerShell
online version: https://learn.microsoft.com/powershell/module/sharepoint-online/Start-SPOSiteOpticalCharacterRecognitionBackfill
applicable: SharePoint Online
title: Start-SPOSiteOpticalCharacterRecognitionBackfill
schema: 2.0.0
author: FarreltinF
ms.author: fanyi
ms.reviewer:
---

# Start-SPOSiteOpticalCharacterRecognitionBackfill

## SYNOPSIS

> [!NOTE]

> > This feature - Optical Character Recognition (OCR) is a pay-as-you-go feature, triggering this cmdlet will incur cost for your organization

Reviewer 2: >This feature -  [](http://example.com/codeflow?start=2&length=15)

"The "...

> > This feature - Optical Character Recognition (OCR) is a pay-as-you-go feature, triggering this cmdlet will incur cost for your organization

Reviewer 2: >This feature - Optical Character Recognition (OCR) is a pay-as-you-go feature, triggering this cmdlet will incur cost for your organization [](http://example.com/codeflow?start=2&length=139)

Optical Character Recognition (OCR) is a pay-as-you-go feature. Running this cmdlet will incur costs for your organization.

> Initiates a job to trigger the Optical Character Recognition (OCR) process for all files for the selected site.

Reviewer 2: >Optical Character Recognition (OCR) [](http://example.com/codeflow?start=31&length=35)

simply use "OCR" going forward, since you already define the acronym in previous line

## SYNTAX

```Powershell

> Start-SPOSiteBackfillOCR [-Site] <string> 

Reviewer 2: this doesn't match the cmdlet name line 13.

also, be sure to use platyPS to generate the template for your documentation.

[https://dev.azure.com/onedrive/ODSP%20Wiki/_wiki/wikis/OSDP-Wiki.wiki/82618/SharePoint-Online-PowerShell-cmdlets?anchor=public-cmdlet-documentation](https://dev.azure.com/onedrive/ODSP%20Wiki/_wiki/wikis/OSDP-Wiki.wiki/82618/SharePoint-Online-PowerShell-cmdlets?anchor=public-cmdlet-documentation)

```

## DESCRIPTION

> This command starts a job to trigger the Optical Character Recognition (OCR) process for files that have either never been OCR processed or have been modified since the last OCR process. This ensures all relevant content in the site is recognized and indexed for improved searchability and accessibility. 

Reviewer 2: >command [](http://example.com/codeflow?start=5&length=7)

cmdlet

> OCR backfill can only be run on OCR-enabled sites. If OCR is disabled for the site, please enable OCR before proceeding with OCR backfilling. Refer to this [article](https://learn.microsoft.com/en-us/microsoft-365/syntex/ocr) for instructions on enabling OCR on the selected site.

Reviewer 2: >https://learn.microsoft.com/en-us/microsoft-365/syntex/ocr [](http://example.com/codeflow?start=166&length=58)

/en-us/microsoft-365/syntex/ocr

## EXAMPLES

### EXAMPLE 1

```powershell

Start-SPOSiteBackfillOCR -Site https://contosoenergy.sharepoint.com/sites/hr
```

> Starts OCR process for all content hasn't been processed before in the selected site.

Reviewer 2: >hasn't [](http://example.com/codeflow?start=35&length=6)

that hasn't

### EXAMPLE 2

```powershell

$site = Get-SPOSIte -Identity https://contosoenergy.sharepoint.com/sites/hr
Start-SPOSiteBackfillOCR -Site $site 
```

> Start OCR process for all content hasn't been processed before in the selected site.

Reviewer 2: you can remove this example -- it's not adding any value

## PARAMETERS

### -Site

Specifies the site URL where OCR process should be enabled on.

```yaml
Type: String
Parameter Sets: Default
Aliases:
Applicable: SharePoint Online

Required: True
Position: 1
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### CommonParameters

This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

## RELATED LINKS

[Overview of optical character recognition in SharePoint](https://learn.microsoft.com/en-us/microsoft-365/syntex/ocr-overview)

> [Learn about optical character recognition in Microsoft Purview](https://learn.microsoft.com/en-us/purview/ocr-learn-about?tabs=purview)

Reviewer 2: we only use relative links, i.e., the stuff after the locale
